### ISSUE TITLE
How to upgrade mattermost-docker version

### ISSUE THREAD

**Issue by craph** [mattermost/mattermost-docker#445]
Hi all,

I write this issue because I'm using mattermost-docker in version : 
```
Mattermost Version: 5.16.3
Database Schema Version: 5.16.0
Database: postgres
```

I would like to upgrade my installed version to the latest one but I can't find the correct way to achieve it in the documentation.

So, what is the correct way to do that ?

Thank you very much.

Best regards,

**Comment by caugner** [comment]
You can find the update instructions here:
https://github.com/mattermost/mattermost-docker#update-mattermost-to-latest-version

**Comment by gpopesc** [comment]
I have a question also related to this topic.
In followed the instructions, but in the in gihtub is still old version
ENV MM_VERSION=5.31.0
It means that the 5.32 is not available for docker?

https://github.com/mattermost/mattermost-docker/blob/master/app/Dockerfile

**Comment by caugner** [comment]
@gpopesc I agree, this is confusing. I used to have `master` checked out, just like you, but it seems you now have to check out one of the release branches, e.g. `release-5.32`.

**Comment by caugner** [comment]
@gpopesc Don't update to `release-5.32` though, it's **broken**: https://github.com/mattermost/mattermost-docker/issues/489

**Comment by simonyipeter** [comment]
Hi,

I've followed this instructions to update the Mattermost:
https://github.com/mattermost/mattermost-docker#update-mattermost-to-latest-version
It was error free, but mobile app still alerting it's an 5.30 version.
Mattermost Version: 5.31.0
Database Schema Version: 5.30.0
Database: postgres

How can I upgrade the database to 5.31?!
Thanks is advance

**Comment by AntoineDavidSupersoniks** [comment]
> Hi,
> 
> I've followed this instructions to update the Mattermost:
> https://github.com/mattermost/mattermost-docker#update-mattermost-to-latest-version
> It was error free, but mobile app still alerting it's an 5.30 version.
> Mattermost Version: 5.31.0
> Database Schema Version: 5.30.0
> Database: postgres
> 
> How can I upgrade the database to 5.31?!
> Thanks is advance

I've the same issue, do you have a solution ?

**Comment by caugner** [comment]
> How can I upgrade the database to 5.31?!

I don't recall having had this problem, but make sure to `docker-compose build` and `docker-compose up -d`, which should migrate the database without problems.
